assets/scripts/segments/Segment.jsx

```diff
@@ -239,8 +239,8 @@ export class Segment extends React.Component {
     // Get localized names from store, fall back to segment default names if translated
     // text is not found. TODO: port to react-intl/formatMessage later.
     const displayName = segment.label || getLocaleSegmentName(segment.type, segment.variantString)
-    const capacity = getSegmentCapacity(segment).capacity.average
-    const showCapacity = enableAnalytics && capacity !== undefined
+    const { capacity: { average, display } } = getSegmentCapacity(segment)
```

Comment:
Why not have this destructure `display` to have a default value of `true`?

```js
const { capacity: { average, display = true } } = getSegmentCapacity(segment)
```

This would prevent having to set it at all in `street_analytics.js`, except when the value is `false`.